docs/contributing/glossary.md

```diff
@@ -275,7 +275,7 @@ We also maintain a [DevOps and Security Glossary](https://www.sumologic.com/glos
 
 **[Scheduled Search](/docs/alerts/scheduled-searches/schedule-search)**. When you save a log search, you can schedule it to run it on a periodic basis, and configure an alert that will issue a notification every time the search runs, or when a condition you specify is met.
 
-**[Scheduled View](/docs/manage/scheduled-views)**. A Scheduled View is a pre-aggregated index of a subset of data. After building a Scheduled View, you'll be able to run queries against that data set. Because the data is pre-aggregated, meaning that query you'll use to create a Scheduled View contains an aggregate function, search results return much quicker. Additionally, queries run against a Scheduled View cannot time out. Queries that run against Views can be used in scheduled searches, Dashboards, and in ad hoc searches.
+**[Scheduled View](/docs/manage/views)**. A Scheduled View is a pre-aggregated index of a subset of data. After building a Scheduled View, you'll be able to run queries against that data set. Because the data is pre-aggregated, meaning that query you'll use to create a Scheduled View contains an aggregate function, search results return much quicker. Additionally, queries run against a Scheduled View cannot time out. Queries that run against Views can be used in scheduled searches, Dashboards, and in ad hoc searches.
```

Comment:
Should this paragraph be changed to reflect Scheduled View --> View?

Reply:
We still have scheduled views but for now it is a part of views. Previously, the user was only able to track the Scheduled views created under the **Manage Data > Logs > Scheduled Views*, which was making the user to loose track of other scheduled details for example, scheduled search. So, currently we have introduced this **Views** page, where the user can get a list of all of the scheduled data under one indexes.